frontleft_fisheye_to_frontright_fisheye:
- Translation: [0.000, -0.061, 0.038]
- Rotation: in Quaternion [-0.521, -0.000, -0.005, 0.853]
- Rotation: in RPY (radian) [-1.097, -0.006, -0.009]
- Rotation: in RPY (degree) [-62.834, -0.341, -0.526]
- Matrix:
    - [ 1.000,  0.009,  0.005,  0.000]
    - [-0.009,  0.457,  0.890, -0.061]
    - [ 0.006, -0.890,  0.457,  0.038]
    - [ 0.000,  0.000,  0.000,  1.000]

body_to_frontleft:
  Matrix:
    - [-0.424,  0.467,  0.776,  0.415]
    - [-0.002,  0.856, -0.516,  0.035]
    - [-0.906, -0.221, -0.362,  0.023]
    - [ 0.000,  0.000,  0.000,  1.000]

body_to_frontleft_fisheye:
  Matrix:
    - [-0.421,  0.466,  0.778,  0.384]
    - [ 0.003,  0.859, -0.512,  0.034]
    - [-0.907, -0.213, -0.363, -0.048]
    - [ 0.000,  0.000,  0.000,  1.000]

frontleft_fisheye_to_frontright_fisheye:
  Matrix:
    - [ 1.000,  0.009,  0.005,  0.000]
    - [-0.009,  0.457,  0.890, -0.061]
    - [ 0.006, -0.890,  0.457,  0.038]
    - [ 0.000,  0.000,  0.000,  1.000]

feet_center_to_frontleft:
  Matrix:
    - [-0.446,  0.464,  0.765,  0.390]
    - [ 0.003,  0.856, -0.517,  0.054]
    - [-0.895, -0.228, -0.383,  0.525]
    - [ 0.000,  0.000,  0.000,  1.000]

feet_Center_to_frontleft_fisheye:
  Matrix:
    - [-0.441,  0.461,  0.770,  0.328]
    - [ 0.005,  0.859, -0.511,  0.038]
    - [-0.898, -0.222, -0.381,  0.454]
    - [ 0.000,  0.000,  0.000,  1.000]

frontleft_depth_intrinsic:
  Matrix:
    - [212.83216857910156, 0.0, 213.3726348876953]
    - [0.0, 212.83216857910156, 116.8132553100586]
    - [0.0, 0.0, 1.0]

frontleft_fisheye_intrinsic:
  Matrix:
    - [330.6578369140625, 0.0, 319.98614501953125]
    - [0.0, 331.0872497558594, 237.93775939941406]
    - [0.0, 0.0, 1.0]